apps/funke/app.config.js

```diff
@@ -52,7 +52,21 @@ const config = {
   updates: {
     fallbackToCacheTimeout: 0,
   },
-  plugins: ['expo-font', 'expo-secure-store', 'expo-router'],
+  plugins: [
+    [
+      'expo-font',
+      {
+        fonts: [
+          '../../node_modules/@expo-google-fonts/raleway/Raleway_400Regular.ttf',
+          '../../node_modules/@expo-google-fonts/raleway/Raleway_500Medium.ttf',
+          '../../node_modules/@expo-google-fonts/raleway/Raleway_600SemiBold.ttf',
+          '../../node_modules/@expo-google-fonts/raleway/Raleway_700Bold.ttf',
```

Comment:
We actually use 'Raleaway' for only the titles and headings. The body font is 'Open sans'.

Reply:
Okay will update 👍

Reply:
I will do this in a separate PR, easier and it's not an issue for the Funke wallet